ui/src/router/components/form/components/RouteDropDownOption.js

```diff
@@ -1,18 +1,22 @@
-import React from "react";
 import {
   EuiBadge,
   EuiFlexGroup,
   EuiFlexItem,
-  EuiText,
   EuiTextColor,
+  EuiText,
   EuiToolTip,
 } from "@elastic/eui";
 
-export const RuleCardRouteDropDownOption = ({ id, endpoint, isDefault }) => {
+export const RouteDropDownOption = ({
```

Comment:
This has been refactored to also let the Fallback Route / Final Response Route panel use the same format for the options (the badge + url) as shown in the PR description.